app/commands/decidim/accountability/admin/create_result.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Accountability
+    module Admin
+      # This command is executed when the user creates a Result from the admin
+      # panel.
+      class CreateResult < Rectify::Command
```

Comment:
Maybe you could inherit from the actual command in `Decidim::Admin` to kind of future-proof it?